### PR TITLE
Eliminate Audacity's spam on stdout and stderr

### PR DESCRIPTION
**Pull request by ksoze95**
Resolves: #2569

*(short description of the changes and the motivation to make the changes)*

- [x] I signed [CLA](https://www.audacityteam.org/cla/)
- [x] The title of the pull request describes an issue it addresses
- [x] If changes are extensive, then there is a sequence of easily reviewable commits
- [x] Each commit's message describes its purpose and effects
- [x] There are no behavior changes unnecessary for the stated purpose of the PR

Recommended:
- [X] Each commit compiles and runs on my machine without known undesirable changes of behavior

### CODE REVIEW

File: src/ProjectManager.cpp [hunk 1]
```diff
@@ -216,6 +216,10 @@ void InitProjectWindow( ProjectWindow &window )
    // Near as I can tell, this is only a problem under Windows.
    //
 
+   //
+   // Create the Menus and Commands
+   //
+   MenuManager::Get( project ).CreateMenusAndCommands( project );
```
Comment (Paul-Licameli) [inline src/ProjectManager.cpp:222]: This is a simple change,  but I would phrase the commit comment to describe the program’s behavior after the change,  not before.

Comment (Paul-Licameli) [reply src/ProjectManager.cpp:222]: Also add a comment in code, why the ordering of these two calls matters.

**Review by Paul-Licameli — CHANGES_REQUESTED** [review]
Comments and other code movement

File: src/Menus.cpp [hunk 2]
```diff
@@ -46,6 +46,7 @@
 
 #include <wx/menu.h>
 #include <wx/windowptr.h>
+#include <wx/log.h>
```
[the remark this answers is not in the document]
Comment (ksoze95) [reply src/Menus.cpp:49]: :)

File: src/ProjectManager.cpp (hunk 1 above)
Comment (ksoze95) [reply src/ProjectManager.cpp]: I have moved function call back to ProjectManager::New, just before the InitProjectWindow.

**Review by Paul-Licameli — APPROVED** [review]

**Review by Paul-Licameli — CHANGES_REQUESTED** [review]

**Review by Paul-Licameli — APPROVED** [review]

**Pull request merged**